packages/manager/modules/anthos/src/dashboard/general-information/order-public-ip/order-public-ip.routing.js

```diff
@@ -0,0 +1,14 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('anthos.dashboard.general-information.order-public-ip', {
+    url: '/order-public-ip',
+    views: {
+      modal: {
+        component: 'orderPublicIp',
+      },
+    },
+    layout: 'modal',
+    resolve: {
+      breacrumb: () => false,
```

Comment:
```suggestion
      breacrumb: () => null,
```

Reply:
Why null not false @JeremyDec ?

Reply:
Hmm in fact, all places where we disable the breadcrumb, set the `breadcrumb` to null, it is more for consistency.

Also I see a typo in the property name: `breacrumb` should be `breadcrumb`.

Reply:
True, fixed also for others files.
thanks